src/MicrosoftTeams.ts

```diff
@@ -588,6 +590,49 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * enable print capability
+   */
+  export function enablePrintCapability(): void {
+    ensureInitialized();
+    printCapabilityEnabled = true;
+    /**
+     * adding ctrl+P and cmd+P handler
+     */
+    document.addEventListener("keydown", function(event: KeyboardEvent): void {
+      if ((event.ctrlKey || event.metaKey) && event.keyCode === 80) {
+        microsoftTeams.print();
+        event.cancelBubble = true;
+        event.preventDefault();
+        event.stopImmediatePropagation();
+      }
+    });
+  }
+
+  /**
+   * Registers a handler for print.
+   * default print handler
+   */
+  export function print(): void {
+    if (printCapabilityEnabled) {
+      if (customPrintHandler) {
+        customPrintHandler();
+      } else {
+        window.print();
+      }
+    }
+  }
+
+  /**
+   * Registers a custom handler for print.
+   * @param handler The handler to invoke when printHandler is called.
+   */
+  export function registerCustomPrintHandler(handler: () => void): void {
```

Comment:
What might an app do in a custom print handler?

Reply:
as discussed offline will remove this support as of now